app/screens/channel_info/channel_info.js

```diff
@@ -308,7 +308,6 @@ export default class ChannelInfo extends PureComponent {
                 passProps: {
                     isPermalink: true,
                     onClose: this.handleClosePermalink,
-                    onPermalinkPress: this.handlePermalinkPress,
```

Comment:
why this got removed? this is to avoid going from a permalink view to a permalink view, right?

Reply:
It's no longer needed since I changed it so that Permalinks and Hashtags do nothing on Permalink view to simplify the navigation